### DISCUSSION TITLE
Background color for alpha part of the image

### DISCUSSION THREAD

**Discussion by LichiMan** [ggarra13/mrv2#32] [Q&A] [answered]
Hi! Hope this kind of questions goes in Q&A.

I've loaded an exr sequence with alpha channel and looking at the Color layer, which has alpha channel, I see the grey background color of the application instead of black.

Is there a way to change the color of the background in the alpha part of an image? I've been looking in every corner of the app including Preferences, but I couldn't find anything.

Thanks!

**Comment by ggarra13** [comment, marked as answer]
Yes, albeit it is somwhatt hidden.  Go to Window->Preferences.  From there, select Lanuguage and Colors.  In the View box, there's a Background square color.  Click on it and a color popup will appear.  Change the color to whatever you like, like full black.
When you exit mrv2, the color will get saved as a preference in $HOME/.filmaura/mrvv2.prefs.

**Reply by LichiMan** [reply]
Sorry! I saw that one but I didn't try it because I thought it was for the background color of the app.
Thank you. And thanks for making mrv2 as an open source software. Also for release binary builds. Thanks!
